src/main/java/com/bisnode/opa/client/data/OpaDocument.java

```diff
@@ -1,9 +1,64 @@
 package com.bisnode.opa.client.data;
 
-import lombok.Value;
+import java.beans.ConstructorProperties;
+import java.util.Objects;
 
-@Value
-public class OpaDocument {
-    String path;
-    String content;
+/**
+ * Class wrapping OPA document content and path to it
+ */
+
+public final class OpaDocument {
+    /**
+     * Path to the document (eg. "this/is/path/to/document")
+     */
+    private final String path;
```

Comment:
Should these docs not be on the public getters rather than on the private fields?

Reply:
Yep, my bad